src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -82,6 +91,10 @@ export const VenueLandingPage: React.FunctionComponent<VenueLandingPageProps> =
         new Date().getTime() / 1000 && event.price > 0
   );
 
+  const showJoinVenueModal = useCallback(() => {
+    setIsJoinVenueModalOpen(true);
+  }, []);
+
```

Comment:
I would tend to put these alongside their corresponding `useState` to keep everything 'grouped together'